src/components/DataEntryModal/DataEntryModal.jsx

```diff
@@ -30,7 +28,25 @@ const DataEntryModal = ({ isOpen, onClose }) => {
     event_id: 0, // Should be changed when connected with CHECKIN
     is_checked_in: false,
   });
-  const [other, setOther] = useState('');
+  // const [other, setOther] = useState('');
+  const other = '';
+  let [unusualItemsArray, setUnusualItemsArray] = useState([]);
+
+  // parses unusual items input - single entries come as strings, multiple entries come as strings of the form {item1, item2... itemn}
+  useEffect(() => {
+    formatUnusualItems();
+  }, []);
```

Comment:
⚠️ **[eslint]** <[react-hooks/exhaustive-deps](https://github.com/facebook/react/issues/14920)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>React Hook useEffect has a missing dependency: 'formatUnusualItems'. Either include it or remove the dependency array.